cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -516,7 +546,9 @@ angular.module(PKG.name + '.commons')
         programType: '@',
         programId: '@',
         runId: '@',
-        getDownloadFilename: '&'
+        getDownloadFilename: '&',
+        integratedWith: '@',
+        statusName: '@'
```

Comment:
This is actually the entity or log name right? ProgramId or app ID or `Cloudera Navigator`.